release_tester/license_manager_tests/base/single_server_base.py

```diff
@@ -22,6 +23,7 @@ def start(self):
         """start a single server setup before running tests"""
         self.start_single_server()
 
+    # pylint: disable=attribute-defined-outside-init
```

Comment:
I also don't like this; rather create a constructor that has the members in question intialized with `None`

Reply:
It is initialized in parent class __init__ method. I don't know why the weirdo keeps complaining.
https://github.com/arangodb/release-test-automation/blob/fb5934300fbac38468304cbfc28481924790390c/release_tester/license_manager_tests/base/license_manager_base_test_suite.py#L55